api/provider/aws/task_create.go

```diff
@@ -1,7 +1,90 @@
 package aws
 
-import "github.com/quintilesims/layer0/common/models"
+import (
+	"fmt"
 
-func (t *TaskProvider) Create(req models.CreateTaskRequest) (*models.Task, error) {
-	return nil, nil
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+func (t *TaskProvider) Create(req models.CreateTaskRequest) (string, error) {
+	taskID := generateEntityID(req.TaskName)
+	fqEnvironmentID := addLayer0Prefix(t.Config.Instance(), req.EnvironmentID)
+
+	deployName, deployVersion, err := lookupDeployNameAndVersion(t.TagStore, req.DeployID)
+	if err != nil {
+		return "", err
+	}
+
+	clusterName := fqEnvironmentID
+	startedBy := t.Config.Instance()
```

Comment:
what does `startedBy` mean?

Reply:
Sorry, I missed the comment in the MR originally...
"Using the startedBy field when creating tasks, and then using that same field to filter tasks during ListTasks."

Reply:
this variable name could be better... possibly a better would be more explicit like `taskStartedBy`

Reply:
that variable matches what is made in the aws call. This is a RoT instance where we explicitly set a new variable to show the relationships between layer0 and AWS concepts.